draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1193,6 +1213,147 @@ In addition, as noted in {{new-quic}}, one of the motivations for using QUIC as
 
 Other QUIC extensions, not yet proposed, might also be useful with RoQ.
 
+# Implementation Status
+
+> **RFC Editor's note:** Please remove this section prior to publication of a
+> final version of this document.
+
+This section records the status of known implementations of the protocol defined
+by this specification at the time of posting of this Internet-Draft, and is
+based on a proposal described in {{?RFC7942}}. The description of
+implementations in this section is intended to assist the IETF in its decision
+processes in progressing drafts to RFCs. Please note that the listing of any
+individual implementation here does not imply endorsement by the IETF.
+Furthermore, no effort has been spent to verify the information presented here
+that was supplied by IETF contributors. This is not intended as, and must not be
+construed to be, a catalog of available implementations or their features.
+Readers are advised to note that other implementations may exist.
+
+According to {{?RFC7942}}, "this will allow reviewers and working groups to
+assign due consideration to documents that have the benefit of running code,
+which may serve as evidence of valuable experimentation and feedback that have
+made the implemented protocols more mature. It is up to the individual working
+groups to use this information as they see fit".
+
+## mengelbart/roq
+
+Ogranization:
+: Technical University of Munich
+
+Implementation:
+: {{roq}}
+
+Descritption:
+: *roq* is a library implementing the basic encapsulation described in
+{{encapsulation}}. The library uses the Go programming language and supports the
+{{quic-go}} QUIC implementation.
+
+Level of Maturity:
+: prototype
+
+Coverage:
+: The library supports sending and receiving RTP and RTCP packets using QUIC
+streams and QUIC DATAGRAMs and multiplexing using flow identifiers. Applications
+using the library are responsible for appropriate signaling, setting up QUIC
+connections, and managing RTP sessions. Applications have to choose whether to
+send RTP and RTCP packets over streams or DATAGRAMs, and applications also have
+control over the QUIC and RTP congestion controllers in use since they control
+the QUIC connection setup and can thus configure the QUIC stack they use to
+their preferences.
+
+Version Compatibility:
+: The library implements {{?I-D.draft-ietf-avtcore-rtp-over-quic-10}}.
+
+Licensing:
+: MIT License
+
+Implementation Experience:
+: The authors have no experience with the following:
+
+* Using the Multipath Extension for QUIC in combination with RoQ.
+* Influence of default priorities of QUIC implementations between streams and
+  DATAGRAMs on the performance of RTCP.
+* DATAGRAMs that are queued (and thus delayed) or dropped on expiration before
+  being transmitted due to congestion.
+* Translating between RoQ and RTP over UDP.
+* Performance impacts of multiplexing many sessions on a single QUIC connection.
```

Comment:
```suggestion
: The implementer reports they have no experience with the following:

* Using RoQ with QUIC connections during path changes, whether due to QUIC connection migration or Multipath Extension for QUIC. As described in {{futures-new-ext}}, we expect future work on this topic.
* Influence of default priorities of QUIC implementations between streams and
  DATAGRAMs on the performance of RTCP. As described in {{futures-impl-deploy}}, we expect future work on this topic.
* DATAGRAMs that are queued (and thus delayed) or dropped on expiration before
  being transmitted due to congestion. As described in {{futures-impl-deploy}}, we expect future work on this topic
* Translating between RoQ and RTP over UDP. As described in {{topologies}}, RoQ can be used to connect to some RTP middleboxes using some topologies, and these middleboxes might be connecting RoQ endpoints and non-RoQ endpoints, so will need to translate between RoQ and RTP over UDP. 
* Performance impacts of multiplexing many RTP sessions on a single QUIC connection. As described in {{futures-impl-deploy}}, we expect future work on this topic
```
So, what I'm thinking is, this section could map pretty closely onto the rest of the document, and especially to {#futures}. If that makes sense, we might use {#futures} as a checklist, to name things that haven't been implemented or tested yet. 

Does that make sense?